### ISSUE TITLE
Vanilla DOOM smooth weapons strange finding with Inter-DOOM

### ISSUE THREAD

**Issue by Meerschweinmann** [JNechaevsky/international-doom#140]
Hi Julia.

I hardly dare to open an issue anymore :)

On my mission these last days to optimize my "vanilla" DOOM experience i have been playing around with your RD skies, widescreen assets, the idkfa soundtrack and some other things i can find.

Surely you know Fraggle's wonderful Vanilla DOOM smooth weapons mod.
If not, you can find it here:

https://www.doomworld.com/forum/topic/102462-vanilla-doom-smooth-weapons/

With this mod i found something i have never seen before. 
In map 09 of DOOM 2 there is a chainsaw in the floor at the exit room with the red star on ground.

I have made a screenshot that you can see what i mean. As i saw this i checked the mod in Woof and it does not happen there.
So unfortunately it seems something ID specific. 

Here is a picture of that mysterious happening.

![DOOM03](https://github.com/JNechaevsky/international-doom/assets/116672360/7773632c-784e-43f0-8866-1dd12a1f5093)

**Comment by JNechaevsky** [comment]
Looks familiar. Have you loaded savegame, that's was written before loading smooth weapons? I'm amraid, despite of 100% demo-compatible, smooth weapons are not very savegame compatible. Also, take a try `TNTEM` cheat code, it will turn monsters that's doesn't have x-death states into ... quite something.

**Comment by Meerschweinmann** [comment]
No, i have not loaded any savegame. As i saw that first i closed ID, did IDCLEV 09 and IDCLIP to show if this chainsaw is gone. I will try the TNTEM cheat code.

**Comment by Meerschweinmann** [comment]
You are right. When i go into that room and do TNTEM, i get that chainsaw in the floor. Then it seems a problem of that mod when i understand you correct?

**Comment by JNechaevsky** [comment]
So, does it happens only after `TNTEM`, or even normally? I don't see it until `TNTEM`:

![image](https://github.com/JNechaevsky/international-doom/assets/21193394/b9973f33-288c-4981-b8da-c6c0c6cf98bc)

![image](https://github.com/JNechaevsky/international-doom/assets/21193394/606bed46-f5bc-46b0-b7b4-3755822d6a38)

Technically, this must be a Dehacked lump problem, but no idea what is happening. But AFAIR it also happens same way in Crispy Doom.

**Comment by Meerschweinmann** [comment]
To be honest i can't reproduce it anymore. Only when running into that room and do TNTEM.

**Comment by Meerschweinmann** [comment]
What does me wonder is that i had it with 7.4 too. But i copied that over my dev-build folder and did not make a fresh folder.

**Comment by Meerschweinmann** [comment]
I have made an absolutely fresh ID folder, copied only DOOM2.WAD in there, changed no options, punched some monsters in that room and even after TNTEM, this chainsaw was not seen.

![DOOM00](https://github.com/JNechaevsky/international-doom/assets/116672360/67e845ca-8403-489a-a60e-deaeb6692877)

**Comment by Meerschweinmann** [comment]
It seems definitive a fault of that weapon mod. Eternity Engine shows the chainsaw too when using the KILLEM cheat, and WOOF crashes with KILLEM in map 09 when the mod is loaded.

Sorry for the wasted time.

**Comment by JNechaevsky** [comment]
So, Guter Herr wants me to have a hearth attack, right before weekly report. And right before weekend. 😅
Yeah, I've seen it before and it happens same way in Crispy Doom, where all current extended Dehacked support comes from. Looks likes it is happening with Pain Elemental, probably because of some very complicated tricks in `DEHACKED` itself.

**Comment by Meerschweinmann** [comment]
Дорогая леди, it was not my intention to give you an heart attack 😊
In the end i was so sceptical to myself that i even checked If there was a save game i had loaded without noticing it, but there was none.
And the TNTEM cheat i did not know before you mentioned it here.
In the end i was just confused how that could happen.
But no matter what, because EE handles IT the same like ID and Woof crashes with KILLEM at map 09, it is definitive the mods fault.
So fortunately everything is fine with ID.

Have you seen what i have found out about Inter-DOOMs max horizontal Resolution and the wipe effect in the issue we discussed before?
